xml/deploy_cephadm.xml

```diff
@@ -797,6 +797,13 @@ config: OK
      valid, it is a good idea to export its configuration to a file:
     </para>
 <screen>&prompt.smaster;ceph-salt export > cluster.json</screen>
+    <warning>
+      <para>
+       Executing the <command>ceph-salt export</command> includes the
```

Comment:
"Executing the `ceph-salt export` command includes the SSH private key" doesn't sound quite right.

How about: "The output of the `ceph-salt export` command includes the SSH private key" ?